### DISCUSSION TITLE
Welcome to Logseq-Git-Sync-101 Discussions!

### DISCUSSION THREAD

**Discussion by charliie-dev** [CharlesChiuGit/Logseq-Git-Sync-101#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Solidwire99** [comment]
Don here... Was in IT from college in 89 till Ynot2K then off to the next most logical thing the home building trade. 
I have the IT Tool chest equivalent of a carpenters tool box left in the weather for 20+ yrs.
A stack of faded & expired certifications for Win NT, Novell Netware & Red Hat Linux. At least one of the 3 exists & looks similar with whopping changes & additional tools.
Anyway...
Just poking around for the moment considering Logseq w/Git ver control.
Also, after seeing the limitations (size constraints) of the Assets folder I was wondering if it's possible (reasonable) to consider allowing the Assets Folder to exist elsewhere? Such as local,  public or  private cloud?
Obviously this would be without the expectation of accessing those links (pics, videos, PDFs, etc...) if local & accessing notes on other device. 
Unfortunately, without this option I can only imagine an abuse of Gits data privileges from a significant percentage of participants. Which will likely end up expensive for all of us... Hope I'm wrong.
Thank you for all your efforts
Don 
SoLidWire

**Reply by charliie-dev** [reply]
Hi Don,

It looks like you have the skill set to do some command line operations! Maybe consider using [git LFS](https://git-lfs.com/)?
I think using git-lfs would alleviate some of your issue.
